<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoMenu</class>
 <widget class="QWidget" name="PhotoMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>456</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>692</width>
    <height>456</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>692</width>
    <height>456</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="video_lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>480</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="make_photo_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>110</width>
     <height>96</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/photo_fw_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/photo_fw.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/photo_fw.png);
     border-width: 3;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="light_on_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>110</width>
     <height>96</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/flash_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/flash.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/flash.png);
     border-width: 3;}
 QPushButton:checked {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/flash_n.png);
     border-width: 3;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="photo_bgrd_lbl">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>456</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding-right: 15px; /* make room for the arrows */
border-image: url(:/Pyramid/fon_next_menu.png);
border-width: 3;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="save_photo_lbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>480</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../Resources/PyramidWidgets.qrc">:/Pyramid/photo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="make_vidio_but">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>110</width>
     <height>96</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"> QPushButton:hover:pressed {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/vidio_fw_n.png);
     border-width: 3;}
 
QPushButton{
     padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/vidio_fw.png);
     border-width: 3;}

 QPushButton:hover {
    padding-right: 15px; /* make room for the arrows */
	border-image: url(:/Pyramid/vidio_fw.png);
     border-width: 3;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="videoStat">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>0</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>photo_bgrd_lbl</zorder>
  <zorder>make_vidio_but</zorder>
  <zorder>make_photo_but</zorder>
  <zorder>light_on_but</zorder>
  <zorder>video_lbl</zorder>
  <zorder>save_photo_lbl</zorder>
  <zorder>videoStat</zorder>
 </widget>
 <resources>
  <include location="../Resources/PyramidWidgets.qrc"/>
  <include location="../Resources/PyramidWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
